docs/source/guide/ses-rules.rst

```diff
@@ -0,0 +1,144 @@
+A rule can send email to other AWS services including but not limited to Amazon S3, Amazon SNS, or AWS Lambda.
+
+For more information, see `Managing Receipt Rule Sets for Amazon SES Email Receiving <http://aws.amazon.com/documentation/ses/receiving-email-managing-receipt-rule-sets.html>`_ and `Managing Receipt Rules for Amazon SES Email Receiving <http://aws.amazon.com/documentation/ses/receiving-email-managing-receipt-rules.html>`_.
+
+The following examples show how to:
+
+* Create a receipt rule set using `create_receipt_rule_set()  <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ses.html#SES.Client.create_receipt_rule_set>`_.
+* Create a receipt rule using `create_receipt_rule() <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ses.html#SES.Client.create_receipt_rule>`_.
+* Remove a receipt rule using `delete_receipt_rule() <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ses.html#SES.Client.delete_receipt_rule>`_.
+* Remove a receipt rule set using `delete_receipt_rule_set() <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ses.html#SES.Client.delete_receipt_rule_set>`_.
+
+Prerequisite Tasks
+==================
+
+To set up and run this example, you must first complete these tasks:
+
+* Configure your AWS credentials, as described in :doc:`quickstart`.
+
+Create a Receipt Rule Set
+==========================
+
+A receipt rule set contains a collection of receipt rules. You must have at least one receipt rule set associated with your account before you can create a receipt rule. To create a receipt rule set, provide a unique RuleSetName and use the `CreateReceiptRuleSet <https://docs.aws.amazon.com/ses/latest/APIReference/API_CreateReceiptRuleSet.html>`_ operation.
+
+Example
+-------
+
+.. code-block:: python
+
+    import boto3
+
+    # Create SES client
+    ses = boto3.client('ses')
+
+    response = ses.create_receipt_rule_set(
+      RuleSetName = 'NAME',
+    )
+
+    print(response)
+
+Create a Receipt Rule
+=====================
+
+Control your incoming email by adding a receipt rule to an existing receipt rule set. This example shows you how to create a receipt rule that sends incoming messages to an Amazon S3 bucket, but you can also send messages to Amazon SNS and AWS Lambda. To create a receipt rule, provide a rule and the RuleSetName to the `CreateReceiptRule <https://docs.aws.amazon.com/ses/latest/APIReference/API_CreateReceiptRule.html>`_ operation.
+
+Example
+-------
+
+.. code-block:: python
+
+    import boto3
+
+    # Create SES client
+    ses = boto3.client('ses')
+
+    response = ses.create_receipt_rule(
+      RuleSetName   = 'RULE_SET_NAME',
+      Rule          = {
+        'Name'      : 'RULE_NAME',
```

Comment:
Same as above comment, this one is trickier than the other since there are a lot of options. Just pick a use case for the sample choose all the options based on that use case. And make sure the final result is copy-paste able valid python.